daemon/containerd/image_save_test.go

```diff
@@ -93,11 +111,19 @@ func TestImageMultiplatformSaveShallowWithoutNative(t *testing.T) {
 		assert.NilError(t, err)
 	})
 	t.Run("export native", func(t *testing.T) {
-		err = imgSvc.ExportImage(ctx, []string{img.Name}, &native, io.Discard)
+		err = imgSvc.ExportImage(ctx, []string{img.Name}, []ocispec.Platform{native}, io.Discard)
 		assert.Check(t, is.ErrorType(err, cerrdefs.IsNotFound))
 	})
 	t.Run("export arm64", func(t *testing.T) {
-		err = imgSvc.ExportImage(ctx, []string{img.Name}, &arm64, io.Discard)
+		err = imgSvc.ExportImage(ctx, []string{img.Name}, []ocispec.Platform{arm64}, io.Discard)
 		assert.NilError(t, err)
 	})
+	t.Run("export multiple platforms", func(t *testing.T) {
+		err = imgSvc.ExportImage(ctx, []string{img.Name}, []ocispec.Platform{arm64, riscv64}, io.Discard)
+		assert.NilError(t, err)
+	})
+	t.Run("export multiple platforms with some missing", func(t *testing.T) {
+		err = imgSvc.ExportImage(ctx, []string{img.Name}, []ocispec.Platform{arm64, native}, io.Discard)
+		assert.Check(t, is.ErrorType(err, cerrdefs.IsNotFound))
+	})
```

Comment:
Can you add a test case that tries to export platform that doesn't exist at all in the image index?

Make sure we error out instead of not saving it silently.

Reply:
Good idea, done.